### ISSUE TITLE
Tracking outdated information 

### ISSUE THREAD

**Issue by ann0see** [jamulussoftware/jamulussoftware.github.io#525]
This issue is to track outdated information on the website (as requested by @jujudusud)
 **This only tracks and should track the english version**
- [x] 1. Jamulus main window screenshot doesn't show the latest changes
- [x] 2. Main window screenshot is outdated
- [x] 3. Commands on the Choosing-a-Server-Type should be jamulus (lower case), not Jamulus (upper case)
- [x] 4. Client-Troubleshooting: Getting blasted by somebody joining your jam with feedback or noise? --> GUI has changed. We need to add that this is an advanced option
- [x] 5. Command line options (a lot of things are undocumented; we're already aware of that)
- [ ] 6. Jamulus Demos (move this to the KB?)
- [x] 7. FAQ: Diagram still uses old icons
- [x] 8. Hardware setup who still knows that this page exists ;-)
- [x] 9. Installation guides: The buttons at the bottom should be named Setup not Onboarding
- [x] 10. Installation for Linux: too short. We should at least hint at the compilation instructions or add a headline about "other distros"
- [x] 11. Running a private server, server win/mac/linux...: Should be included in an upcoming server guide
- [x] 12. Manual: Feedback Protection should be explained differently: Attempts to detect audio feedback loops or loud noise in the first three seconds after you connected to a server. Once detected, this feature will activate the "Mute Myself" button and mute you in your own mix. An error message will tell you that you should fix the problem (or similar)
- [x]  13. Manual: We're not consistent with systemctl vs service. The former is more modern.
- [x]  14. ~~We still use Central Server: https://jamulus.io/de/wiki/Server-Win-Mac#other-options~~ Upddated with SrvAdm manual in next release.

These are the issues I found from my first round through the website.

**Comment by jujudusud** [comment]
Commenting for now only:
Why did jamulus should be lower case ?
Yes we should put "Jamulus Demos" in KB
FAQ Diagrams you can use FR screenshots images (with translation work in EN) (the public one needs to be identical to private in the client screen)

**Comment by ann0see** [comment]
> Why did jamulus should be lower case ?

since the .deb file installs the binary as `jamulus` not `Jamulus`

**Comment by jujudusud** [comment]
Ok, but it is not the standard build option. Anyway ... I'm ok with this explanation. Disapointed but ok ...
Is Jamulus also with lower case in Windows and macOS ? In Archlinux it is using the Jamulus standard upper case since 3.5.6 release. Do I have to revert this to go with lower case ?

**Comment by ann0see** [comment]
In Windows it‘s upper case. Not sure about macOS

**Comment by gegeweb** [comment]
> Ok, but it is not the standard build option. Anyway ... I'm ok with this explanation. Disapointed but ok ...
> Is Jamulus also with lower case in Windows and macOS ? 

Windows and macOS do not work like linux.
Windows file system is case insensitive.
macOS too by default.

Under macOS the binaries are in the *.app (Jamulus.app and JamulusServer.app, it is uppercase).

I agree about the missing compilation information for Linux (information was in the previous version of the doc…).
I use Jamulus (server and client) on a Rpi4 and there is no *.deb for armhf or arch64 at this time and no up to date package in the Buster and Bullseye repositories.

**Comment by gegeweb** [comment]
About choosing server type page, I still think that it is necessary to explain the context of use of the different types of servers.
Including a central server.

In order to answer :
Do I need a server?
What do I need it for? Private use for my group or public use for jamming with others?

For the central server it can be useful for an organization (music schools, online events...).

**Comment by ann0see** [comment]
Agree. Can you make a proposal PR to the next-release branch?

**Comment by jujudusud** [comment]
> About choosing server type page, I still think that it is necessary to explain the context of use of the different types of servers.

@gegeweb -> Isn't it already here ? https://jamulus.io/wiki/Running-a-Server

**Comment by gilgongo** [comment]
11 is covered in my Monster PR From Hell here https://github.com/jamulussoftware/jamuluswebsite/pull/527 

On point 8, I was going to do a separate PR for that as it now overlaps on some information in Setup and elsewhere, and may not in fact be very useful now. It's based on a very old text Volker wrote on the SF website in fact.

**Comment by gilgongo** [comment]
@gegeweb I incorporated suggestions made in https://github.com/jamulussoftware/jamuluswebsite/issues/45 in #527

**Comment by gilgongo** [comment]
Although it doesn't count as "outdated" I'd also like to have a screenshot of the main window on the home page, replacing the cloud image, as I think that's perhaps a bit more "engaging".

**Comment by gilgongo** [comment]
On point 10, we're trying to get people to use binary packages so we can support them better. We also shouldn't imply we maintain things for other distros (most of which are very out of date as far as I can tell). And anyone who has any interest in compiling will just git clone and read COMPILING.md. 

So I think expanding that page may cause problems over the longer term. The fact that we don't have ARM binaries means we should probably point Rpi users to the repo perhaps - but that's as far as I'd go.

**Comment by jujudusud** [comment]
My point of view is that we should provide a Raspberry Pi os arm package since it is nearly the same as Debian x86_64.
Convince users to use binaries and not changing the help pages since compiling info are on the repository.
Another thing is that we don't want anyone to run a server since there are hundreds of theme available.

**Comment by gilgongo** [comment]
BTW is this ticket a (partial) duplicate? https://github.com/jamulussoftware/jamulus/issues/1779

**Comment by gilgongo** [comment]
@jujudusud In fact I see there is this re. ARM: https://github.com/jamulussoftware/jamulus/issues/1452

**Comment by gilgongo** [comment]
@DavidSavinkoff Was it you that found some issue with port forwarding of public servers? I was wondering what the second sentence in this item in the server troubleshooting was all about:

https://jamulus.io/wiki/Server-Troubleshooting#can-i-convert-my-private-server-to-being-a-public-one

**Comment by ann0see** [comment]
> BTW is this ticket a (partial) duplicate? jamulussoftware/jamulus#1779

Kind of. However there’s also some stuff not in sync the other way round. Especially concerning our language guide

**Comment by ghost** [comment]
The information here is Not mine:
https://jamulus.io/wiki/Server-Troubleshooting#can-i-convert-my-private-server-to-being-a-public-one
> Was it you that found some issue with port forwarding of public servers?
> 
No.
> I was wondering what the second sentence in this item in the server troubleshooting was all about.
> 
I don't know because this in Not my comment.


The information here Is mine:
https://jamulus.io/wiki/Tips-Tricks-More#converting-a-public-server-to-a-private-one-on-the-fly

**Comment by gilgongo** [comment]
@DavidSavinkoff OK thanks, sorry to bother you.

**Comment by ann0see** [comment]
We still use Central Server: https://jamulus.io/de/wiki/Server-Win-Mac#other-options

**Comment by gilgongo** [comment]
I've added that as no. 14 in the list.  EDIT: actually that page is going as per the Server Admin Manual changes, so got updated with that.

**Comment by gilgongo** [comment]
@ann0see re. no.5 what's missing?

**Comment by ann0see** [comment]
You can check by looking at main.cpp starting here: https://github.com/jamulussoftware/jamulus/blob/e0cac75ee0d5ef0a6f85f6b88367abab9b659853/src/main.cpp#L113

Since It’s not yet live, we‘d need to check the page based on next-release.

**Comment by gilgongo** [comment]
Sorry, I meant I can't see anything missing from there after I added the missing stuff reported here https://github.com/jamulussoftware/jamulus/issues/1837. ~~Does your "outdated info" list refer to stuff that's currenly on release or next-release?~~ You say "a lot of things are undocumented; we're already aware of that"

**Comment by ann0see** [comment]
Yes, I think it referred to release and the PR you mentioned

**Comment by ann0see** [comment]
According to https://github.com/jamulussoftware/jamulus/discussions/1903#discussioncomment-986273 -s is missing on the live site. Since it’s an important option, how should we proceed? Add it to the live page?

**Comment by gilgongo** [comment]
We should add it for the sake of completeness in the list of command line options, although until now I don't think anyone's noticed since it's part of the server setup instructions, and so is effectively documented that way. Maybe just add it to the admin manual in https://github.com/jamulussoftware/jamuluswebsite/pull/527 and wait for it to be translated.

**Comment by gilgongo** [comment]
-s is in the new Server Manual. There were a number of small errors and ommissions I fixed in that.

**Comment by ann0see** [comment]
Ok. Just waiting on the next release then?

**Comment by gilgongo** [comment]
I think so.

**Comment by ignotus666** [comment]
I'd be in favour of doing a translation update before the next release, even if it's limited to a couple of languages, for a couple of reasons. First, so that at least the EN (and ES + FR?) documentation is up to date with the current state of the application and not a step behind. And second, to test the new translation workflow and see how it works out in practice, instead of waiting until the next release is imminent and potentially running into problems. Is there anything in 'next-release' that shouldn't go live before the next release?

No hurry though, there's a lot of stuff that needs to be done before it can happen, but simply throwing the idea out there. I wouldn't want to find out just a few weeks before the release that the po4a setup is catastrophically flawed.

**Comment by ann0see** [comment]
@jamulussoftware/maindevelopers what do you think?

Actually I had a similar thought
